client/src/App.jsx

```diff
@@ -5,7 +5,7 @@ import { MuiThemeProvider, CssBaseline } from "@material-ui/core";
 import { BrowserRouter, Route, Redirect } from "react-router-dom";
 import { theme } from "./themes/theme";
 import AuthPage from "./pages/AuthPage";
-import Dashboard from "./pages/Dashboard";
+import Dashboard from "./pages/Dashboard/index";
```

Comment:
let's be consistent with naming conventions - I more commonly see with React the name of the files are the name of the components and are PascalCase